scanner/cmd/scannerctl/main.go

```diff
@@ -91,23 +91,26 @@ func rootCmd(ctx context.Context) *cobra.Command {
 		}
 		// Set options for the gRPC connection.
 		opts := []client.Option{
-			client.WithAddress(*address),
-			client.WithServerName(*serverName),
+			client.WithIndexerAddress(*indexerAddr),
+			client.WithIndexerServerName(*indexerServerName),
+			client.WithIndexerSubject(mtls.ScannerV4IndexerSubject),
+			client.WithMatcherAddress(*matcherAddr),
+			client.WithMatcherServerName(*matcherServerName),
+			client.WithMatcherSubject(mtls.ScannerV4MatcherSubject),
 		}
 		if *skipTLSVerify {
 			opts = append(opts, client.SkipTLSVerification)
 		}
-		if *indexerAddr != "" {
-			opts = append(opts, client.WithIndexerAddress(*indexerAddr))
-		}
-		if *matcherAddr != "" {
-			opts = append(opts, client.WithMatcherAddress(*matcherAddr))
+		if *address != "" {
+			opts = append(opts, client.WithAddress(*address))
```

Comment:
nit: This makes the less specific `--address` have precedence over the more specific `--indexer-address` and `--matcher-address` - usually see the 'more specific' option have precedence.

Reply:
I'm just going to remove `--address` to make life easier and punt this to the next person who may value `--address` enough. I think setting each individually may be a pain, but also livable